src/example-types/terms_stats.js

```diff
@@ -1,12 +1,23 @@
 var _ = require('lodash/fp')
+let F = require('futil-js')
 var esTwoLevel = require('./esTwoLevelAggregation').result
 let { buildRegexQueryForWords } = require('../regex')
 let { getField } = require('../fields')
+let { metrics, hasValidMetrics } = require('../aggUtils')
 
 module.exports = {
   validContext: context => context.key_field && context.value_field,
   async result(context, search, schema) {
     let field = getField(schema, context.key_field, context.fieldMode)
+    let orderPaths = F.arrayToObject(
```

Comment:
there's no need to generate this whole array, just inline the key calculation - it'll be less code, easier to follow, and more efficent